share/picongpu/tests/XrayDiffractionCells/validate.ipynb

```diff
@@ -0,0 +1,4975 @@
+{
```

Comment:
Please load this notebook once in jupiter and clear the cash. This will remove all currently base64 encoded images in this file.

Reply:
Yes, this notebook is not finished at all actually, nor was it in the original PR.

Reply:
I remember that @ax3l was strictly against including `*.ipynb`. I am fine with adding a notebook as long as the cache is cleared. But ideally deriving just a `*.py` from the notebook (via the save as option) would be better if we want to have a text based, human readable versioning for later changes.

Reply:
Thanks, I can do that. Do you know if we have examples for other plugins?

Reply:
So far we avoided this - therefore there are no examples in the code, only some code snippets in the documentation.

Reply:
I'd be against uploading an `*.ipynb`. Let's stay with a pure `*.py`. 
These can be run via scripts and deliver clear output if the validation was successful or not.

I like Jupyter notebooks but I would refrain from adding them to the code now.

Reply:
@sbastrakov I thought now a lot if we should add the notebook and I agree with @n01r. 
One very strong reason to remove the notebook is that nearly impossible to review the content and we will be soon not able to take care that the code still works.

Reply:
Okay, notebook removed, will add a `.py`.